doc/en/user/source/configuration/contact.rst

```diff
@@ -3,7 +3,7 @@
 Contact Information
 ===================
 
-The Contact Information is used in the ``Capabilities``` document of the WMS and other web services, and is publicly accessible.
+The Contact Information is used in the ``Capabilities`` document of the WMS and other web services, and is publicly accessible.
```

Comment:
missing backticks, oughta be  \`\`\`Capabilities\`\`\`

Reply:
2 backticks is correct: 
<img width="753" alt="image" src="https://github.com/user-attachments/assets/f2fb8dec-c367-4a44-a6c5-c10e8c09cfb6" />